_episodes/03-lists.md

```diff
@@ -172,8 +172,8 @@ does not.
 >
 > Here is a visual example of how indexing a list of lists `x` works:
 >
-> <a href='https://twitter.com/hadleywickham/status/643381054758363136'>
-> ![The first element of a list. Adapted from @hadleywickham's tweet about R lists.](../fig/indexing_lists_python.png)</a>
+> [![The first element of a list.
+   Adapted from @hadleywickham.](../fig/indexing_lists_python.png)][hadleywickham-tweet]
```

Comment:
I know this does not change much, but let's add `> ` in the beginning of this line.